src/crimson/os/seastore/btree/fixed_kv_btree.h

```diff
@@ -1297,6 +1297,17 @@ class FixedKVBtree {
       ceph_assert(depth == meta.depth);
       ceph_assert(begin == meta.begin);
       ceph_assert(end == meta.end);
+      if (unlikely(ret->get_in_extent_checksum()
+          != ret->get_last_committed_crc())) {
+        SUBERRORT(
+          seastore_fixedkv_tree,
+          "internal fixedkv extent checksum inconsistent, "
+          "recorded: {}, actually: {}",
+          c.trans,
+          ret->get_in_extent_checksum(),
+          ret->get_last_committed_crc());
+        ceph_abort();
+      }
```

Comment:
Probably better to move before all the other checks in case of device corruption ?